openpype/tools/settings/settings/widgets.py

```diff
@@ -640,11 +679,27 @@ def __init__(self, parent):
         layout.addWidget(label_widget, 0)
         layout.addWidget(project_list, 1)
 
+        if only_active:
+            inactive_chk = None
```

Comment:
Option n2 for SyncServer fix
```suggestion
        if only_active:
            inactive_chk = None
            project_proxy.set_filter_enabled(False)
```